### PR TITLE
210 fix typos in codemarkdownreadme gp

### PR DESCRIPTION
**Pull request by potvingab**
I fixed some broken links, corrected some English typos, replaced all the "Solara" I could see by "East Horizon", and tried to improve the website by making the titles and descriptions more consistent.

### CODE REVIEW

File: vignettes/AssuranceConsecutiveStudies.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Bayesian Assurance Using East-R"
+title: "Bayesian Assurance of Consecutive Studies"
```
Comment (Anoopsinghrawat) [inline vignettes/AssuranceConsecutiveStudies.Rmd:2]: @kwathen, can you please confirm if this renaming is correct?

File: vignettes/Home.Rmd
Comment (Anoopsinghrawat) [inline vignettes/Home.Rmd:1]: @potvingab , why did we delete this file?

File: pkgdown/index.md
Comment (Anoopsinghrawat) [inline pkgdown/index.md:1]: @potvingab , it looks like this file was newly added. Can you tell us its purpose?

File: vignettes/Home.Rmd
Comment (potvingab) [reply vignettes/Home.Rmd:1]: @Anoopsinghrawat If I'm not mistaken, this file is not used by pkgdown at all. I think it was used in the past, but now README.md (root directory) is used to build the home page. If you look at the content, it seems pretty much outdated (only 5 examples when CyneRgy has way more now), that's why I deleted it.

File: vignettes/AssuranceConsecutiveStudies.Rmd (hunk 1 above)
Comment (potvingab) [reply vignettes/AssuranceConsecutiveStudies.Rmd:2]: Just to clarify: I tried to make consistent the titles over all pages of CyneRgy's website, but I did not invent any new name. The name "Bayesian Assurance of Consecutive Studies" was from the Examples list. It seemed like the best to keep between the two.

Comment (kwathen) [reply vignettes/AssuranceConsecutiveStudies.Rmd:2]: It should be Bayesian Assurance of Consecutive Studies

File: vignettes/Home.Rmd
Comment (kwathen) [reply vignettes/Home.Rmd:1]: Fine to delete this file

File: pkgdown/index.md
Comment (potvingab) [reply pkgdown/index.md:1]: Kyle told me the best was probably to keep both files (README.md and the new index.md) for all links to work correctly, but add comments to remind us to make edits in both files if we want to change the home page.

**Comment by Anoopsinghrawat** [comment]
No more comments from me. @potvingab, there seem to be some merge conflicts now. Can you please resolve them. Kyle can then merge to dev if there are no further changes required.

**Comment by potvingab** [comment]
I resolved the conflicts @Anoopsinghrawat. It should be ready to merge @kwathen.

**Pull request merged**